src/index.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable */
+// import "./sdToSeqD";
+import checkSD from './validateSd';
+import { SQSD } from '../definitions/defs';
+import parseSD from './parseSD';
+import generateTS from './codeGen';
+// import generateSD from "./generateSD";
```

Comment:
Same as above

Reply:
`// import generateSD from "./generateSD"` could also be removed. I left it there explicitly as to show how missing functionality could eventually be implemented in the same way (in this case, SD generation from SeqD). We can safely remove it though..